ntu_learn_downloader/parsing.py

```diff
@@ -62,10 +62,13 @@ def is_file(child, img):
             link = hyperlink.get("href")
             result.append(SLecture(name.strip(), link))
         elif is_file(c, img):
-            hyperlink = c.find("a")
-            name = hyperlink.text
-            link = hyperlink.get("href")
-            result.append(SDoc(name.strip(), link))
+            try :
+                hyperlink = c.find("a")
```

Comment:
I feel better with a `if hyperlink is None` guard to make the no href tag present issue more explicit but this is fine

Reply:
@raynoldng agreed, this was a quick fix. Tied up right now, I can PR something better tomorrow.

Reply:
Its okay, I can do that. Let me merge your PR in so that you at least get cognition. Thanks for implementing the fix